src/services/fictions.ts

```diff
@@ -113,6 +113,7 @@ export class FictionsService {
      * @returns - Array of search fiction blurbs.
      */
     public async search(keyword: string, page: number = 1) {
+        //TODO: change/fix
```

Comment:
this comment is useless, what needs to be changed/fixed and why?

either remove or create an issue and link it